scripts/dl_cleanup.py

```diff
@@ -90,6 +97,7 @@ def parseVer_GIT(match, filepath):
 
 versionRegex = (
 	(re.compile(r"(.+)[-_](\d+)\.(\d+)\.(\d+)\.(\d+)"), parseVer_1234),	# xxx-1.2.3.4
+	(re.compile(r"(.+)[-_](\d\d\d\d)-?(\d\d)-?(\d\d)-"), parseVer_ymd_GIT_SHASUM),	# xxx-YYYY-MM-DD-GIT_SHASUM
```

Comment:
Cool, do you have an example package I could test this with?

Reply:
Ok found one example why this is required
Example package is
openwrt-keyring

```
Without the additional regex the version detected is
expat 36894051140322197504
openwrt-keyring-2021-02-20 90905565772833481555968
libubox 37280870617400523882496
With the additional regex
expat 36894051140322197504
openwrt-keyring 37280870336002856583168
libubox 37280870617400523882496
```

This cause a problem in detecting the version of the package as the package is detected as a separate one with the date.

Reply:
Think i should add this to the commit description?